guide/interactions/autocomplete.md

```diff
@@ -0,0 +1,110 @@
+# Autocomplete
+
+Slash command options provide the feature of autocomplete. In this section we'll cover how to both add autocomplete support to your command and also how to respond to it.
```

Comment:
- Slash command options provide the feature of autocomplete.
Maybe this sentence could be rephrased? And maybe tell the reader what Autocomplete is and why and where they should use it.

```suggestion
Slash command options provide the feature of autocomplete. In this section, we'll cover how to add autocomplete support to your command and how to respond to it.
```